src/twisted/web/_newclient.py

```diff
@@ -644,6 +640,77 @@ def _ensureValidURI(uri):
     raise ValueError(f"Invalid URI {uri!r}")
 
 
+def _decint(data: bytes) -> int:
```

Comment:
this looks… like a good idea… but uh, why is it in the *client* implementation? doesn't the server need to do this too?